### PR TITLE
Case-insensitive email

### PR DESCRIPTION
**Pull request by ch3njust1n**
# Description

/email, /signup, /forgot, and /reset use transformation pipes to make email lowercase

## Checklist

- [x] This PR can be reviewed in under 30 minutes
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have assigned reviewers to this PR.

### CODE REVIEW

File: packages/server/src/auth/auth.controller.ts [hunk 1]
```diff
@@ -37,11 +48,13 @@ export class RecoveryController {
   constructor(private authService: AuthService) {}
 
   @Post('forgot')
+  @UsePipes(new ForgotPasswordTransformPipe())
   async forgotPassword(@Body() user: ForgotDto): Promise<void> {
     return this.authService.forgotPassword(user.projectId, user.email);
   }
 
   @Post('reset')
+  @UsePipes(new ResetPasswordTransformPipe())
```
Comment (cbolles) [inline packages/server/src/auth/auth.controller.ts:57]: The resolver will also need the transform added to it

**Review by cbolles — CHANGES_REQUESTED** [review]
Looks good, but the gql resolvers will also need the modification added in

**Review by gregfrasco — DISMISSED** [review]

File: packages/server/src/auth/auth.controller.ts (hunk 1 above)
Comment (ch3njust1n) [reply packages/server/src/auth/auth.controller.ts:57]: What's the resolver?

Comment (cbolles) [reply packages/server/src/auth/auth.controller.ts:57]: The GraphQL resolver which handles the GraphQL requests (`auth.resolver`). In NestJS you should be able to use the pipes the same way on resolver methods

**Review by gregfrasco — APPROVED** [review]

**Review by cbolles — APPROVED** [review]

**Pull request merged**